src/client/client.ts

```diff
@@ -55,10 +55,16 @@ export class AuthlessClient {
    * extractOrSave(authlessResponse)
    * ```
    */
-  async fetch (params: FetchParams): Promise<any> {
+  async fetch (params: FetchParams): Promise<IResponse> {
```

Comment:
```suggestion
  async fetch (params: FetchParams): Promise<Response> {
```

This should be a `Response` instance, so I can call `response.toResources()` etc.